packages/wdio-utils/src/initializeServices.ts

```diff
@@ -95,9 +95,9 @@ export async function initializeLauncherService (
     config: Omit<WebdriverIO.Config, 'capabilities' | keyof Services.HookFunctions>,
     caps: Capabilities.TestrunnerCapabilities
 ): Promise<{
-    ignoredWorkerServices: string[];
-    launcherServices: Services.ServiceInstance[];
-}> {
+        ignoredWorkerServices: string[];
+        launcherServices: Services.ServiceInstance[];
+    }> {
```

Comment:
This is a weird formatting issue, I've seen other PRs wanting to introduce this.
```suggestion
    ignoredWorkerServices: string[];
    launcherServices: Services.ServiceInstance[];
}> {
```